### ISSUE TITLE
ENUM_MARKET_CONDITION' - declaration without type ea-input.mqh

### ISSUE THREAD

**Issue by kenorb** [EA31337/EA31337-classes#58]
> ENUM_MARKET_CONDITION' - declaration without type ea-input.mqh 96 8

This errors in ME build 5.0.0.2122, but not in 5.0.0.1966.

**Comment by kenorb** [comment]
Since the new build of MQL5 compilator, class enums are now needs to be explicitly referenced, e.g. `Condition::ENUM_MARKET_CONDITION` instead of `ENUM_MARKET_CONDITION`. This makes the syntax not back compatible.

For example:

MQL4:

```
struct A {
   enum bla {
      A, B, C
   };
};

bla x = C;
```

MQL5:

```
struct A {
   enum bla {
      A, B, C
   };
};

bla x = A::bla::C;
```

The solution is to move the enums outside of the class.

**Comment by kenorb** [comment]
Fixed in #60
